assets/js/googlesitekit/data/index.js

```diff
@@ -51,6 +52,6 @@ Data.collectReducers = collectReducers;
 Data.collectResolvers = collectResolvers;
 Data.collectSelectors = collectSelectors;
 Data.collectState = collectState;
-Data.collectReducers = collectReducers;
```

Comment:
Noting here for future reviewers that this was removed because it's a repeat assignment—it's already on line 52.